src/Services/Kros.Authorization.Api/Controllers/UsersController.cs

```diff
@@ -15,28 +17,50 @@ namespace Kros.Authorization.Api.Controllers
     /// </summary>
     [Route("api/[controller]")]
     [ApiController]
-    [Authorize(AuthenticationSchemes = JwtAuthorizationHelper.JwtSchemeName) ]
+    [Authorize(AuthenticationSchemes = JwtAuthorizationHelper.JwtSchemeName)]
     public class UsersController : ControllerBase
     {
         private readonly IUserService _userService;
 
+        private readonly IPermissionService _permissionsService;
+
         /// <summary>
         /// Constructor.
         /// </summary>
         /// <param name="userService">User service.</param>
-        public UsersController(IUserService userService)
-            => _userService = userService;
+        /// <param name="permissionService">Permission service.</param>
+        public UsersController(IUserService userService, IPermissionService permissionService)
+        {
+            _userService = Check.NotNull(userService, nameof(userService));
+            _permissionsService = Check.NotNull(permissionService, nameof(permissionService));
+        }
 
         /// <summary>
         /// Is user admin?
         /// </summary>
-        /// <returns>Return <see langword="true"/>, if user is admin.</returns>
+        /// <returns>Returns <see langword="true"/>, if user has admin privileges for organization.</returns>
         [HttpGet(nameof(IsAdmin))]
         [ProducesResponseType(200, Type = typeof(bool))]
         public bool IsAdmin()
-        {
-            return _userService.IsAdminFromClaims(User);
-        }
+            => _permissionsService.IsAdminFromClaims(User);
+
+        /// <summary>
```

Comment:
It's ok for this demo, but for a real REST api it's not good to have methods like this. From the point of view of REST, it would be right to have endpoint like `/users/{userId}/roles`.